rpcd/playbooks/ansible.cfg

```diff
@@ -1,5 +1,8 @@
 [defaults]
 gathering = smart
+fact_caching = jsonfile
```

Comment:
The ceph roles heavily relies on fact caching.

We could fact cache at the beginning of each playbook, but it won't be as efficient as the controlled manner implemented in this playbook.

On top of that we heavily rely on deleting facts in a surgical way, which is easy when you rely on json files dumped into the filesystem: you can just delete the file. It's more readable at the end.